global.json

```diff
@@ -1,5 +1,5 @@
 {
-  "msbuild-sdks": {
-    "MSBuild.Sdk.Extras": "1.6.55"
+  "sdk": {
+    "version": "3.0.100-preview-010046"

```

Comment:
There is no 3.0.100 010046 to download from https://dotnet.microsoft.com/download/dotnet-core/3.0